app/src/main/java/com/jerboa/feat/BlurNsfwTypes.kt

```diff
@@ -0,0 +1,28 @@
+package com.jerboa.feat
+
+import android.os.Parcelable
+import androidx.annotation.StringRes
+import com.jerboa.R
+import com.jerboa.datatypes.types.PostView
+import com.jerboa.toEnum
+import kotlinx.parcelize.Parcelize
+
+@Parcelize
+enum class BlurNsfwTypes(@StringRes val resId: Int) : Parcelable {
+    DoNotBlur(R.string.app_settings_do_not_blur),
+    BlurEverywhere(R.string.app_settings_blur_everywhere),
+    BlurEverywhereExceptNsfw(R.string.app_settings_blur_everywhere_except_nsfw),
+    BlurOnlyNsfwCommunity(R.string.app_settings_blur_only_nsfw_community);
+}
+
+fun BlurNsfwTypes.needNsfwBlur(postView : PostView) =
+    this.needNsfwBlur(postView.community.nsfw, postView.post.nsfw)
+
+fun BlurNsfwTypes.needNsfwBlur(isCommunityNsfw: Boolean, isPostNsfw: Boolean = isCommunityNsfw): Boolean {
+    return isPostNsfw && when(this){
+        BlurNsfwTypes.DoNotBlur -> false
```

Comment:
Possible to change this to:
- Nothing
- All
- OnlyInsideNsfwCommunities
- OnlyOutsideNsfwCommunities

These are shorter and more descriptive, we already know they blur from the EnumName

Change the translation to fit these new names too.